package.json

```diff
@@ -25,7 +25,8 @@
     "db:update-client": "npx prisma generate",
     "spec:generate": "cross-env NODE_ENV=development ts-node -r dotenv/config scripts/generateOpenApi.ts && npm run spec:validate",
     "spec:validate": "cross-env NODE_ENV=development ts-node -r dotenv/config scripts/validateOpenApi.ts",
-    "prepublishOnly": "npm run build"
+    "prepublishOnly": "npm run build",
+    "generateJwt": "cross-env NODE_ENV=development ts-node scripts/generateJwt.ts"
```

Comment:
general naming convention used for commands in template is `jwt:generate` (other than for standard npm hooks).

Reply:
fixed